packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -186,7 +186,9 @@ module.exports = function(env) {
 								loader: 'sass-loader',
 								options: {
 									sourceMap: true,
-									includePaths: [...nodeModules],
```

Comment:
A thought: it might be interesting to see if we could provide `includePaths` for those still using node-sass@7 by defining includePaths using `Object.defineProperty(obj, 'includePaths', { value: [...nodeModules] })` so it's non-enumerable?

Reply:
I made the change and tested with sass-loader 7 and 8.  It looks that work well